### PR TITLE
Docsite: clarify a changelog entry format

### PR DESCRIPTION
**Pull request by Andersson007**
##### SUMMARY
Establish a certain changelog entry format:
```
- The change of the module.                               <- doesn't determine its scope, no reference to an issue / PR
- module - The description of the change                  <- starts with a capital letter
- module - the description of the change (https://...)    <- no dot at the end
- module - the description of the change (https://...).   <- this one fully corresponds to the examples
- The description of the change (https://...).            <- also correct, when it affects all modules, for example
- other variations
```

##### ISSUE TYPE
- Docs Pull Request

##### COMPONENT NAME
`docs/docsite/rst/community/development_process.rst`

### CODE REVIEW

**Comment by Andersson007** [comment]
cc @acozine @samccann

File: docs/docsite/rst/community/development_process.rst [hunk 1]
```diff
@@ -184,7 +184,25 @@ A single changelog fragment may contain multiple sections but most will only con
...
+Most changelog entries are ``bugfixes`` or ``minor_changes``.
+
+Changelog fragment entry format
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+When writing a changelog entry, follow the format:
+
+.. code-block:: yaml
+
+  - scope - description starting with a lowercase letter and ending with a dot (https://github.com/reference/to/an/issue or, if there is no issue, reference to a pull request itself).
```
Comment (felixfontein) [inline docs/docsite/rst/community/development_process.rst]:
```suggestion
  - scope - description starting with a lowercase letter and ending with a dot at the very end. Multiple sentences are allowed (https://github.com/reference/to/an/issue or, if there is no issue, reference to a pull request itself).
```

File: docs/docsite/rst/community/development_process.rst [hunk 2]
```diff
@@ -184,7 +184,25 @@ A single changelog fragment may contain multiple sections but most will only con
...
+
+The scope is usually a module or plugin name or group of modules or plugins, for example, ``lookup plugins``.
```
Comment (felixfontein) [inline docs/docsite/rst/community/development_process.rst]:
```suggestion
The scope is usually a module or plugin name or group of modules or plugins, for example, ``lookup plugins``. While module names can (and should) be mentioned directly (``foo_module``), plugin names should always be followed by the type (``foo inventory plugin``).
```

File: docs/docsite/rst/community/development_process.rst [hunk 3]
```diff
@@ -184,7 +184,25 @@ A single changelog fragment may contain multiple sections but most will only con
...
+
+For changes that are not really scoped (for example, which affect a whole collection), follow the format:
+
+.. code-block:: yaml
+
+  - Description starting with an uppercase letter and ending with a dot (https://github.com/reference/to/an/issue or, if there is no issue, reference to a pull request itself).
```
Comment (felixfontein) [inline docs/docsite/rst/community/development_process.rst]:
```suggestion
  - Description starting with an uppercase letter and ending with a dot at the very end. Multiple sentences are allowed (https://github.com/reference/to/an/issue or, if there is no issue, reference to a pull request itself).
```

**Comment by Andersson007** [comment]
@felixfontein i like the suggestions. All committed, thanks! PTAL

File: docs/docsite/rst/community/development_process.rst [hunk 4]
```diff
@@ -184,7 +184,25 @@ A single changelog fragment may contain multiple sections but most will only con
 
 Each changelog entry must contain a link to its issue between parentheses at the end. If there is no corresponding issue, the entry must contain a link to the PR itself.
 
-Most changelog entries will be ``bugfixes`` or ``minor_changes``. When writing a changelog entry that pertains to a particular module, start the entry with ``- [module name] -`` and the following sentence with a lowercase letter.
+Most changelog entries are ``bugfixes`` or ``minor_changes``.
+
+Changelog fragment entry format
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+When writing a changelog entry, follow the format:
```
Comment (samccann) [inline docs/docsite/rst/community/development_process.rst]:
```suggestion
When writing a changelog entry, use the following format:
```

File: docs/docsite/rst/community/development_process.rst [hunk 5]
```diff
@@ -184,7 +184,25 @@ A single changelog fragment may contain multiple sections but most will only con
...
+
+.. code-block:: yaml
+
+  - scope - description starting with a lowercase letter and ending with a dot at the very end. Multiple sentences are allowed (https://github.com/reference/to/an/issue or, if there is no issue, reference to a pull request itself).
```
Comment (samccann) [inline docs/docsite/rst/community/development_process.rst]:
```suggestion
  - scope - description starting with a lowercase letter and ending with a period at the very end. Multiple sentences are allowed (https://github.com/reference/to/an/issue or, if there is no issue, reference to a pull request itself).
```

File: docs/docsite/rst/community/development_process.rst [hunk 6]
```diff
@@ -184,7 +184,25 @@ A single changelog fragment may contain multiple sections but most will only con
...
+
+The scope is usually a module or plugin name or group of modules or plugins, for example, ``lookup plugins``. While module names can (and should) be mentioned directly (``foo_module``), plugin names should always be followed by the type (``foo inventory plugin``).
+
+For changes that are not really scoped (for example, which affect a whole collection), follow the format:
```
Comment (samccann) [inline docs/docsite/rst/community/development_process.rst]:
```suggestion
For changes that are not really scoped (for example, which affect a whole collection), use the following format:
```

File: docs/docsite/rst/community/development_process.rst (hunk 5 above)
Comment (Andersson007) [reply docs/docsite/rst/community/development_process.rst]: didn't know that the "period" is more suitable, thanks:)
committed

**Comment by Andersson007** [comment]
@samccann thanks for the suggestions! All committed. PTAL

**Review by felixfontein — APPROVED** [review]

**Review by samccann — APPROVED** [review]

**Comment by samccann** [comment]
Thanks @Andersson007  (tho I had to google what PTAL meant :-)

**Comment by Andersson007** [comment]
@felixfontein @samccann thanks for reviewing!:)

**Pull request merged**